exercises/dnd-character/description.md

```diff
@@ -0,0 +1,34 @@
+For a game of [Dungeons & Dragons][DND], each player starts by generating a
+character they can play with. This character has, among other things, six
+abilities; strength, dexterity, constitution, intelligence, wisdom and
+charisma. These six abilities have scores that are determined randomly. You
+do this by rolling four 6-sided dice and record the sum of the highest three
+dice.  You do this six times, once for each ability.
+
+Your character's initial hitpoints are 10 + your character's constitution
+modifier. You find your character's constitution modifier by subtracting 10
+from your character's constitution, divide by 2 and round down.
+
+Write a random character generator that follows the rules above.
+
+Normally you would assign each dice roll to an ability freely, but for this
+exercise, assign the dice rolls randomly among the six abilities.
+
+For example, the six dice rolls may look like:
```

Comment:
```suggestion
For example, the 24 dice rolls may look like:
```

Reply:
I think this would be confusing and ambiguous. You're not allowed to combine your 24 dice freely, and you roll them in six groups of four.

Also, shouldn't it be "die rolls" here, too, then?

If I'm rolling four dice six times, is it six die rolls or six dice rolls? It is not conceptually 24 die rolls, since the "largest 3 of 4" rule apply to groups of dice.

Reply:
I could call it "six throws of four dice"?

Reply:
Heh, six throws of four dice does solve the ambiguity. Otherwise, I think you're right: it would be die rolls here also.

Reply:
Thanks. :)

Reply:
I like the six throws of four dice suggestion